charms/kfp-api/src/charm.py

```diff
@@ -243,6 +251,25 @@ def _check_container_connection(self, container: Container) -> None:
         if not container.can_connect():
             raise ErrorWithStatus("Pod startup is not complete", MaintenanceStatus)
 
+    def _check_status(self):
+        """Check status of workload and set status accordingly."""
+        self._check_leader()
+        container = self.unit.get_container(self._container_name)
+        if container:
+            try:
+                check = container.get_check("kfp-api-up")
+            except ModelError as error:
+                raise GenericCharmRuntimeError(
+                    "Failed to run health check on workload container"
+                ) from error
+            if check.status != CheckStatus.UP:
```

Comment:
```suggestion
                ) from error
                
            if check.status != CheckStatus.UP:
```

Reply:
This one failed linting. Removing whitespace.